app/src/main/java/com/jnj/vaccinetracker/common/data/database/entities/ParticipantEntity.kt

```diff
@@ -11,11 +11,14 @@ data class ParticipantEntity(
     override val participantUuid: String,
     @ColumnInfo(collate = ColumnInfo.NOCASE)
     override val participantId: String,
+    override val nin: String?,
```

Comment:
We need to add Nin validation to ensure a user is entering a right Nin. I have a pr that you might need to look into. Well it's not yet finished but you can glance at it and guide me accordingly. Thanks

Reply:
@sherrif10 Let's merge this one, then rebase yours and merge yours.